packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -133,6 +142,11 @@ export default {
             default: true
         },
 
+        hasBackButton: {
```

Comment:
IMO this has the potential to be a bit confusing. If someone passes in `hasCloseButton` and `hasBackButton`, the expected end result isn't obvious; two buttons, or one?

I would prefer keeping the existing `hasCloseButton` but renaming this prop to something like `closeButtonStyle`, which controls whether or not the icon is rendered as a cross or a chevron.

Reply:
It will only display one or the other. If it has a back button the close button won't be displayed. I do see what you mean though, will take a look.

Reply:
Fixed.